R/ep_wp_calculators.R

```diff
@@ -39,15 +39,16 @@
 #' @importFrom stats predict
 #' @export
 calculate_expected_points <- function(pbp_data) {
+
+   # drop existing values of ep and the probs before making new ones
+  pbp_datata <- pbp_data %>% dplyr::select(-tidyselect::any_of(drop.cols))
```

Comment:
This `pbp_datata` is a typo right?

Reply:
Man it was late when I did that. It's a typo, sorry!

Reply:
fixed